### PR TITLE
update zh-trans:content/zh/docs/setup/production-environment/windows/…

### PR DESCRIPTION
**Pull request by tanjunchen**
ref:https://github.com/k8smeetup/website-tasks/issues/3695
/cc @tengqm

### CODE REVIEW

File: content/zh/docs/setup/production-environment/windows/user-guide-windows-containers.md [hunk 1]
```diff
@@ -233,12 +233,136 @@ By adding a taint to all Windows nodes, nothing will be scheduled on them (that
...
+    operator: Equal
+    value: "windows"
+```
+
+<!--
+1. Run `kubectl create -f runtimeClasses.yml` using as a cluster administrator
+1. Add `runtimeClassName: windows-2019` as appropriate to Pod specs
+-->
+1. 集群管理员运行 `kubectl create -f runtimeClasses.yml` 操作
+1. 根据 Pod 规范添加 `runtimeClassName: windows-2019`
```
Comment (tengqm) [inline content/zh/docs/setup/production-environment/windows/user-guide-windows-containers.md]:
```suggestion
1. 根据需要向 Pod 规约中添加 `runtimeClassName: windows-2019`
```

**Comment by tanjunchen** [comment]
/cc @tengqm 
done
/cc @xichengliudui 
/cc @SataQiu

File: content/zh/docs/setup/production-environment/windows/user-guide-windows-containers.md [hunk 2]
```diff
@@ -233,12 +233,136 @@ By adding a taint to all Windows nodes, nothing will be scheduled on them (that
...
+    key: os
+    operator: Equal
+    value: "windows"
+```
+
+<!--
+1. Run `kubectl create -f runtimeClasses.yml` using as a cluster administrator
+1. Add `runtimeClassName: windows-2019` as appropriate to Pod specs
+-->
+1. 集群管理员运行 `kubectl create -f runtimeClasses.yml` 操作
```
Comment (SataQiu) [inline content/zh/docs/setup/production-environment/windows/user-guide-windows-containers.md]: 编号不连续，需要修复下！

图片

Comment (markthink) [reply content/zh/docs/setup/production-environment/windows/user-guide-windows-containers.md]: 问题不算太大,后期可以再优化..

**Pull request merged**